tsconfig.cjs.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "./tsconfig.base.json",
+  "compilerOptions": {
+    "module": "commonjs",
+    "outDir": "./dist/cjs",
+    "target": "ES5"
+  },
+  "include": [
+    "dist/cjs/src/**/*"
+  ],
+  // XXX: if we don't specify this, tsc adds the value of `outDir` to `exclude`
```

Comment:
So if this comes from our custom config, I'd rather take PR correctly setup (probably we should simply copy tsconfig itself when we copy src) or either just maintain current setup. 

I am in favor of make config simpler but feeling this should be included PR to correclty amend with our build pipeline.